### PR TITLE
Fb 2023.07.25

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_7.txt [hunk 1]
```diff
@@ -415,7 +415,7 @@ The Acting Presbyter briefly grinds his teeth together in frustration, and then
 -- どうせロア狙いだろ？　こっちも同じだが、用があるのはヤツの工房だけだ。本体はテメェに任せてやるよ」
 // 本体 here is a bit tricky to translate, still not certain how to best phrase this.
 You're after Roa, yeah?
- Same here, but all I care about is the guy's Workshop.
+ Same here, but all I care about is the guy's workshop.
```
Comment (rschlaikjer) [inline script/Ciel/Day 7/07_01_CIEL07_7.txt:418]: Assuming this was actually intentional, Petrikow?

File: script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 2]
```diff
@@ -112,7 +112,7 @@ Kohaku-san gives me a wide grin.
 -- Page 92, Offset 4188.
 -- もともとむこうで使ってたのは有間の家のものだったから、自分の物なんて着てる服ぐらいで……」
 Everything I used over there originally belonged to the Arima family.
- Pretty much all I own is the clothes on my back..."
+ Pretty much all I own are the clothes on my back..."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 3/03_00_ARC03_2B.txt:115]: `is` is definitely like 'wrong' grammar but weirdly it feels correct to me? I also get more (9k to 3k) google strict hits for `is` over `are` with a phrase search.

File: script/Ciel/Day 7/07_01_CIEL07_7.txt (hunk 1 above)
Comment (Petrikow) [reply script/Ciel/Day 7/07_01_CIEL07_7.txt:418]: Not sure if it was even me, as it appears this term missed the spreadsheet AND the glossary. I'd probably prefer it uncapitalized, honestly.

File: script/Ciel/Day 8/08_01_CIEL08_4.txt [hunk 3]
```diff
@@ -641,8 +641,8 @@ Senpai is my upperclassman at school.
 [sha:051efd8db744d4be60a1b2a7dbb8b6d4004af964]{
 -- Page 381, Offset 19206.
 -- 志貴は騙されているだけよ。埋葬機関の人間はみんな好戦的で、隙あらば襲いかかってくるんだから」
-She's just pulled the wool over your eyes, Shiki.
  Every single member of the Burial Agency is a battle-hungry maniac, they'll pounce at the first opportunity."
+She has just been pulling the wool over your eyes, Shiki.
```
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_4.txt:645]: I feel like this change sort of makes sense in how you present causality in English, but the んだから comes across pretty badly here regardless (makes these feel less connected than they're supposed to be)

>She's just pulling the wool over your eyes, Shiki.
>Don't you get it? All members of the burial agency are battle-hungry maniacs just waiting for the first opportunity to pounce.

Something along those lines would slightly bind them together better and solve the issue more decisively.

**Comment by rschlaikjer** [comment]
Re: the was/were changes, I believe both where referring to hypotheticals, in which case were (subjunctive) is correct

File: script/Arcueid/Day 3/03_00_ARC03_2B.txt (hunk 2 above)
Comment (yk1512) [reply script/Arcueid/Day 3/03_00_ARC03_2B.txt:115]: prefer latter also

**Review by yk1512 — APPROVED** [review]

**Review by Avuxo — APPROVED** [review]

**Pull request merged**